scripts/st2bootstrap-deb.sh

```diff
@@ -339,17 +339,17 @@ configure_st2_user () {
     sudo useradd stanley
   fi
 
-  sudo mkdir -p /home/stanley/.ssh
+  sudo mkdir -p ~stanley/.ssh
```

Comment:
Ah, good point about st2.package.conf. Not much point fixing it in the bootstrap script if it promptly breaks after install. I'm actually not sure how `~` would be interpreted by st2 when it finds it in the config file. I know how the shell interprets it, but it could be different if it's the st2actionrunner process.

@Kami That is explicitly meant to be `~stanley`, so it translates as "stanley's home directory". The other one there `~/stanley` would mean "a file/folder named 'stanley' in the home directory of whichever users is running this command". What I was trying to do in this change is to account for those environments where `/home` is not the default base path for users. Some people instead use things like `/users`. That make sense?